skaffold.yaml

```diff
@@ -10,4 +10,7 @@ deploy:
   kubectl:
     manifests:
     - ops/k8s/deployments/api-deployment.yaml
+    flags:
+      global:
+        - --namespace=data-repo
```

Comment:
remove hard coded namespace

Reply:
There's not a way to templatize this currently -- this is required because we removed the hardcode from api-deployment.yaml. It is the equivalent of saying "when you run kubectl apply or create, make sure to include --namespace=data-repo on the command line".